osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -197,7 +197,7 @@ private void setDistances(double clockRate)
 
                 float distanceBetweenStartPositions = (BaseObject.StackedPosition * scalingFactor - lastObject.StackedPosition * scalingFactor).Length;
 
-                if (MinimumJumpDistance < distanceBetweenStartPositions)
+                if (MinimumJumpDistance < distanceBetweenStartPositions && LazyJumpDistance >= distanceBetweenStartPositions)
```

Comment:
This looks like increasing slider length by 1px can nuke pp, because lazy jump is slightly lower than normal distance now